### PR TITLE
Adding test for KustoIngestClient

### PR DESCRIPTION
**Pull request by t-liyari**

### CODE REVIEW

File: azure-kusto-ingest/tests/input/identitytokenresult.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	"Tables": [{
+		"TableName": "Table_0",
+		"Columns": [{
+			"ColumnName": "AuthorizationContext",
+			"DataType": "String"
+		}],
+		"Rows": [["authorization_context"]]
+	}]
+}
```
Comment (toshetah) [inline azure-kusto-ingest/tests/input/identitytokenresult.json]: Minor: please add a new line

File: azure-kusto-ingest/tests/input/ingestionresourcesresult.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		["TempStorage",
+		"https://storageaccount.blob.core.windows.net/tempstorage?sas"],
+		["TempStorage",
+		"https://storageaccount.blob.core.windows.net/tempstorage?sas"],
+		["TempStorage",
+		"https://storageaccount.blob.core.windows.net/tempstorage?sas"],
+		["IngestionsStatusTable",
+		"https://storageaccount.table.core.windows.net/ingestionsstatus?sas"]]
+	}]
+}
```
Comment (toshetah) [inline azure-kusto-ingest/tests/input/ingestionresourcesresult.json]: Minor: please add a new line

File: azure-kusto-ingest/tests/test_kusto_ingest_client.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+def mocked_requests_post(*args, **kwargs):
+    """Mock to replace requests.post"""
+
+    class MockResponse:
+        """Mock class for KustoResponse."""
+
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.text = str(json_data)
```
Comment (toshetah) [inline azure-kusto-ingest/tests/test_kusto_ingest_client.py]: https://stackoverflow.com/questions/18034272/python-str-vs-unicode-types

This is why we use six.
You can look at the KustoClient tests as an example

Comment (t-liyari) [reply azure-kusto-ingest/tests/test_kusto_ingest_client.py]: Changing  here and also in other places that 'str' appears in our code.

**Pull request merged**